src/altair/beacon_block.rs

```diff
@@ -39,9 +40,9 @@ pub struct BeaconBlock<
     const MAX_VOLUNTARY_EXITS: usize,
     const SYNC_COMMITTEE_SIZE: usize,
 > {
-    #[serde(with = "crate::serde::as_string")]
+    #[cfg_attr(feature = "serde", serde(with = "crate::serde::as_string"))]
```

Comment:
Ypu removed the serde feature in the toml file, there's no way to enable this itf it's needed